<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert1</class>
 <widget class="QWidget" name="insert1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>770</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起始货币</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目标货币</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>汇率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>310</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	
	background-color: qradialgradient(spread:reflect, cx:0.24214, cy:0.864, radius:2, fx:0, fy:1, stop:0.0438596 rgba(242, 165, 194, 255), stop:0.311404 rgba(182, 146, 167, 255), stop:0.574561 rgba(113, 155, 166, 241), stop:0.872807 rgba(150, 106, 161, 255));
}</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
